CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,5 +1,17 @@
 # Changelog 1.11
 
+## 1.11.1 - 2023-04-19
+
+### Changed
+
+- Document the limit of maximum receiver group number on a Linux Node. ([#4850](https://github.com/antrea-io/antrea/pull/4850), [@ceclinux])
+
+### Fixed
+
+- Fix a few bugs during Service update processing. ([#4845](https://github.com/antrea-io/antrea/pull/4845), [@tnqn])
```

Comment:
@tnqn I see the most of bug fixes in #4845 are for internal usage, should I highlight anything from the issue.

Reply:
They are not for internal usage. They can really happen when updating Services. We could say "Fix Service not being updated correctly when stickyMaxAgeSeconds or InternalTrafficPolicy is updated."

Reply:
Updated.